bundles/com.espressif.idf.ui/src/com/espressif/idf/ui/update/InstallToolsHandler.java

```diff
@@ -59,13 +68,16 @@ protected IStatus run(IProgressMonitor monitor)
 				new ExportIDFTools().runToolsExport(pythonExecutablenPath, gitExecutablePath, console);
 				monitor.worked(1);
 				console.println(Messages.InstallToolsHandler_ConfiguredBuildEnvVarMsg);
-				
+
 				monitor.setTaskName(Messages.InstallToolsHandler_AutoConfigureToolchain);
 				configureToolChain();
 				monitor.worked(1);
 				console.println(Messages.InstallToolsHandler_ConfiguredCMakeMsg);
-				
+
 				console.println(Messages.InstallToolsHandler_ToolsCompleted);
+
+				copyOpenOcdRules();
```

Comment:
Can we place this code right after autoconfiguring the tools step? 
```
configureToolChain();
monitor.worked(1);
copyOpenOcdRules();
```

Reply:
Done